applications/luci-app-banip/htdocs/luci-static/resources/view/banip/firewall_log.js

```diff
@@ -1,41 +1,4 @@
 'use strict';
-'require view';
-'require poll';
-'require fs';
+'require tools.views as views';
 
-return view.extend({
-	load: function () {
-		return Promise.all([
-			L.resolveDefault(fs.stat('/sbin/logread'), null),
-			L.resolveDefault(fs.stat('/usr/sbin/logread'), null)
-		]);
-	},
-	render: function (stat) {
-		var logger = stat[0] ? stat[0].path : stat[1] ? stat[1].path : null;
-		poll.add(function () {
-			return L.resolveDefault(fs.exec_direct(logger, ['-e', ' banIP/'])).then(function (res) {
-				var log = document.getElementById("logfile");
-				if (res) {
-					log.value = res.trim();
-				} else {
-					log.value = _('No banIP related firewall logs yet!');
-				}
-				log.scrollTop = log.scrollHeight;
-			});
-		});
-		return E('div', { class: 'cbi-map' },
-			E('div', { class: 'cbi-section' }, [
-				E('div', { class: 'cbi-section-descr' }, _('The syslog output, prefiltered for banIP-related firewall log entries only.')),
-				E('textarea', {
-					'id': 'logfile',
-					'style': 'width: 100% !important; padding: 5px; font-family: monospace',
-					'readonly': 'readonly',
-					'wrap': 'off',
-					'rows': 25
-				})
-			]));
-	},
-	handleSaveApply: null,
-	handleSave: null,
-	handleReset: null
-});
+return views.LogreadBox(" banIP/", "banIP firewall logs");
```

Comment:
` banIP/` it this correct that it starts with space?

Reply:
yep, that's intentional/required

Reply:
looks confusing, but hope it's only for me. Minor style issue: use `'` instead of `"`